axelor-bank-payment/src/main/resources/apps/init-data/en/bankpayment_bankStatementQuery.csv

```diff
@@ -9,11 +9,11 @@
+42;"Equal amount and dates";1;"moveLine?.currencyAmount.abs().equals(currencyAmount) && (moveLine?.date?.equals(operationDate) || moveLine?.date?.equals(valueDate) || moveLine?.dueDate?.equals(operationDate) || moveLine?.dueDate?.equals(valueDate))";1;3
+43;"Equal amount and date within ranges";1;"moveLine?.currencyAmount.abs().equals(currencyAmount) && ((moveLine?.dueDate <= valueDate?.plusDays(%date) && moveLine?.dueDate >=valueDate?.minusDays(%date)) || (moveLine?.date <= valueDate?.plusDays(%date) && moveLine?.date >=valueDate?.minusDays(%date)) || (moveLine?.dueDate <= operationDate?.plusDays(%date) && moveLine?.dueDate >=operationDate?.minusDays(%date)) || (moveLine?.date <= operationDate?.plusDays(%date) && moveLine?.date >=operationDate?.minusDays(%date)))";1;4
+44;"Amount and mode of payment equal and date within the ranges";1;"moveLine?.move?.paymentMode.equals(paymentMode) && moveLine?.currencyAmount.abs().equals(currencyAmount) && ((moveLine?.dueDate <= valueDate?.plusDays(%date) && moveLine?.dueDate >=valueDate?.minusDays(%date)) || (moveLine?.date <= valueDate?.plusDays(%date) && moveLine?.date >=valueDate?.minusDays(%date)) || (moveLine?.dueDate <= operationDate?.plusDays(%date) && moveLine?.dueDate >=operationDate?.minusDays(%date)) || (moveLine?.date <= operationDate?.plusDays(%date) && moveLine?.date >=operationDate?.minusDays(%date)))";1;5
+45;"Equal amount and date within ranges";1;"moveLine?.currencyAmount.abs().equals(currencyAmount) && ((moveLine?.dueDate <= valueDate?.plusDays(%date) && moveLine?.dueDate >=valueDate?.minusDays(%date)) || (moveLine?.date <= valueDate?.plusDays(%date) && moveLine?.date >=valueDate?.minusDays(%date)) || (moveLine?.dueDate <= operationDate?.plusDays(%date) && moveLine?.dueDate >=operationDate?.minusDays(%date)) || (moveLine?.date <= operationDate?.plusDays(%date) && moveLine?.date >=operationDate?.minusDays(%date)))";2;6
+46;"Equal amount";1;"moveLine?.currencyAmount.abs().equals(currencyAmount)";2;7
+47;"Amount and date within ranges";1;"moveLine?.currencyAmount.abs()>= currencyAmount * %amt- && moveLine?.currencyAmount.abs()<= currencyAmount * %amt+ && moveLine?.currencyAmount.abs()>= currencyAmount * %amt- && moveLine?.currencyAmount.abs()<= currencyAmount * %amt+ && ((moveLine?.dueDate <= valueDate?.plusDays(%date) && moveLine?.dueDate >=valueDate?.minusDays(%date)) || (moveLine?.date <= valueDate?.plusDays(%date) && moveLine?.date >=valueDate?.minusDays(%date)) || (moveLine?.dueDate <= operationDate?.plusDays(%date) && moveLine?.dueDate >=operationDate?.minusDays(%date)) || (moveLine?.date <= operationDate?.plusDays(%date) && moveLine?.date >=operationDate?.minusDays(%date)))";3;8

```

Comment:
in groovy we can use `==` instead of `equals()`. also `equals()` will check that the scales are equals (not `==`) and  I don't think we want that.